client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -43,7 +45,7 @@ function getChecklistCompletionProgress( tasks: Task[] | null ) {
 	return Math.round( ( totalCompletedTasks / tasks.length ) * 100 );
 }
 
-const Sidebar = ( { sidebarURL, siteSlug, submit, goNext, goToStep }: SidebarProps ) => {
+const Sidebar = ( { sidebarDomainObject, siteSlug, submit, goNext, goToStep }: SidebarProps ) => {
```

Comment:
I'm passing the domain object instead of the URL, so I can extract the domain URL and status from the same object.